src/main/java/seedu/address/storage/JsonAdaptedClient.java

```diff
@@ -88,72 +97,81 @@ public Client toModelType() throws IllegalValueException {
         for (JsonAdaptedTag tag : tagged) {
             clientTags.add(tag.toModelType());
         }
+        final Set<Note> clientNotes = new HashSet<>();
+        for (JsonAdaptedNote note : this.clientNotes) {
+            clientNotes.add(note.toModelType());
+        }
 
         if (name == null) {
-            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Name.class.getSimpleName()));
+            throw new IllegalValueException(
+                String.format(MISSING_FIELD_MESSAGE_FORMAT, Name.class.getSimpleName()));
         }
         if (!Name.isValidName(name)) {
             throw new IllegalValueException(Name.MESSAGE_CONSTRAINTS);
         }
         final Name modelName = new Name(name);
 
         if (phone == null) {
-            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Phone.class.getSimpleName()));
+            throw new IllegalValueException(
+                String.format(MISSING_FIELD_MESSAGE_FORMAT, Phone.class.getSimpleName()));
```

Comment:
Not too sure why these were reformatted, should probably set ur auto-formatter to reformat only when chars > 120 in a line, same for everything below. Also, I think the second line should be 8 spaces instead of 4, when doing a linebreak

Reply:
Yeap autoformatter :') Changed it.